src/connectdlg.cpp

```diff
@@ -923,3 +933,31 @@ void CConnectDlg::DeleteAllListViewItemChilds ( QTreeWidgetItem* pItem )
         delete pCurChildItem;
     }
 }
+
+void CConnectDlg::contextMenuRequest()
+{
+    // add line to context menu, is dynamic, made at each request
+    QMenu* menu = cbxServerAddr->lineEdit()->createStandardContextMenu();
```

Comment:
As in `src/clientsettingsdlg.cpp`.  Perhaps define the menu in `src/util.cpp`, or somewhere common to both client and server, and use the same one.